management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -120,7 +120,7 @@ public static double getExpectedValueFromRatioHistory(String statName, Context c
         int mostRecentIndex = ratioHistory.getValue().length - 1;
         value = ratioHistory.getValue()[mostRecentIndex].getValue();
       }
-    } while (!Thread.currentThread().isInterrupted() && value != expectedResult);
+    } while (!Thread.currentThread().isInterrupted() && !value.equals(expectedResult));
```

Comment:
@mathieucarbou looks like your suggestion may have worked.  I tested it locally and the build just passed.  Although the build took an hour!